global_helpers/global_helpers_test.py

```diff
@@ -177,37 +180,37 @@ def test_additional_details_str_list_json(self):
 
 
 class TestTorExitNodes(unittest.TestCase):
+
+    event = {"p_enrichment": {"tor_exit_nodes": {"foo": {"ip": "1.2.3.4"}}}}
+
+    # match against array field
+    event_list = {
+        "p_enrichment": {
+            "tor_exit_nodes": {"p_any_ip_addresses": [{"ip": "1.2.3.4"}, {"ip": "1.2.3.5"}]}
+        }
+    }
+
     def test_ip_address_not_found(self):
         """Should not find anything"""
         tor_exit_nodes = p_tor_h.TorExitNodes({})
         ip_address = tor_exit_nodes.ip_address("foo")
         self.assertEqual(ip_address, None)
 
-    def test_has_exit_nodes_found(self):
+    def test_ip_address__found(self):
```

Comment:
```suggestion
    def test_ip_address_found(self):
```
extra underscore in this method

Reply:
Fixed